Firefly III - Wrong Php version required is 8.5 php version installed 8.4.18

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

Firefly III

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/firefly.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

pve-manager/9.1.5/80cf92a64bef6889 (running kernel: 6.17.9-1-pve)

### 📝 Provide a clear and concise description of the issue.

  ✔️   Customized LXC Container
  ✔️   Disabled IPv6
  ✔️   Set up Container OS
  ✔️   Network Connected: redacted
  ✔️   IPv4 Internet Connected
  ✖️   IPv6 Internet Not Connected
  ✔️   Git DNS: github.com:(✔️ ) raw.githubusercontent.com:(✔️ ) api.github.com:(✔️ ) git.community-scripts.org:(✔️ )
  ✔️   Updated Container OS
  ✔️   Setup PHP 8.4
  ✔️   Setup Composer
  ✔️   Setup MariaDB 11.8.3 (distribution packages)
  ✔️   Set up MariaDB Database
  ✔️   Deployed: firefly (6.5.0)
  ✔️   Deployed: dataimporter (2.1.1)
  ✖️   in line 31: exit code 1 (General error / Operation not permitted)
 → composer install --no-dev --no-plugins --no-interaction

  --- Last 20 lines of log ---
 28/37 [=====================>------]  75%
 37/37 [============================] 100%
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
Composer detected issues in your platform:

Your Composer dependencies require a PHP version ">= 8.5.0". You are running 8.4.18.

Script Illuminate\Foundation\ComposerScripts::postAutoloadDump handling the post-autoload-dump event terminated with an exception
Plugins have been disabled automatically as you are running as root, this may be the cause of the following exception. See also https://getcomposer.org/root

In platform_check.php line 22:
                                                                                                                                   
  Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.5.0". You are running 8.4.18.  
                                                                                                                                   

install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--download-only] [--dev] [--no-suggest] [--no-dev] [--no-security-blocking] [--no-autoloader] [--no-progress] [--no-install] [--audit] [--audit-format AUDIT-FORMAT] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>...]

[2026-02-23 23:16:43] [ERROR] in line 31: exit code 1 (General error / Operation not permitted)
[2026-02-23 23:16:43] composer install --no-dev --no-plugins --no-interaction
  -----------------------------------
  📋 Full log: /root/.install-1d97012d.log

  ✖️   Installation failed in container 106 (exit code: 1)


full log is not generated in that position

### 🔄 Steps to reproduce the issue.

start the command 
bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/firefly.sh)"

from the node shell

### ❌ Paste the full error output (if available).

✖️ in line 31: exit code 1 (General error / Operation not permitted)
→ composer install --no-dev --no-plugins --no-interaction

--- Last 20 lines of log ---
28/37 [=====================>------] 75%
37/37 [============================] 100%
Generating optimized autoload files

    Illuminate\Foundation\ComposerScripts::postAutoloadDump
    Composer detected issues in your platform:

Your Composer dependencies require a PHP version ">= 8.5.0". You are running 8.4.18.

Script Illuminate\Foundation\ComposerScripts::postAutoloadDump handling the post-autoload-dump event terminated with an exception
Plugins have been disabled automatically as you are running as root, this may be the cause of the following exception. See also https://getcomposer.org/root

In platform_check.php line 22:

Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.5.0". You are running 8.4.18.

install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--download-only] [--dev] [--no-suggest] [--no-dev] [--no-security-blocking] [--no-autoloader] [--no-progress] [--no-install] [--audit] [--audit-format AUDIT-FORMAT] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [...]
[2026-02-23 23:16:43] [ERROR] in line 31: exit code 1 (General error / Operation not permitted)
[2026-02-23 23:16:43] composer install --no-dev --no-plugins --no-interaction

📋 Full log: /root/.install-1d97012d.log

✖️ Installation failed in container 106 (exit code: 1)

### 🖼️ Additional context (optional).

_No response_

--- a/ct/firefly.sh
+++ b/ct/firefly.sh
@@ -28,7 +28,10 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
+
   setup_mariadb
+  PHP_VERSION="8.5" PHP_APACHE="YES" setup_php
+
   if check_for_gh_release "firefly" "firefly-iii/firefly-iii"; then
     systemctl stop apache2
     cp /opt/firefly/.env /opt/.env

--- a/install/firefly-install.sh
+++ b/install/firefly-install.sh
@@ -13,7 +13,7 @@ setting_up_container
 network_check
 update_os
 
-PHP_VERSION="8.4" PHP_APACHE="YES" setup_php
+PHP_VERSION="8.5" PHP_APACHE="YES" setup_php
 setup_composer
 setup_mariadb
 MARIADB_DB_NAME="firefly" MARIADB_DB_USER="firefly" setup_mariadb_db

